### DISCUSSION TITLE
Checkout of private submodules is broken

### DISCUSSION THREAD

**Discussion by tuler** [community/community#25828] [Actions]
<p>I think some upcoming changes with <code>actions/checkout</code> is breaking the current behavior of private submodules checkout. All my builds a broken.</p>
<p>An issue is already open.</p>
<p><a href="https://github.com/actions/checkout/issues/60" rel="nofollow noopener">https://github.com/actions/checkout/issues/60</a></p>

**Comment by thboop** [comment, marked as answer]
<p>Hello <a class="mention" href="/tuler">@tuler</a> ,</p>
<p>I’m happy to report the <code>uses: actions/checkout@v1</code> behavior around private submodules has been fixed! Please let me know if you see any more issues. </p>
